Neos.ContentRepository.Core/Classes/Feature/Common/EmbedsContentStreamAndNodeAggregateId.php

```diff
@@ -14,18 +14,14 @@
 
 namespace Neos\ContentRepository\Core\Feature\Common;
 
-use Neos\ContentRepository\Core\SharedModel\Node\NodeAggregateId;
-use Neos\ContentRepository\Core\SharedModel\Workspace\ContentStreamId;
-
 /**
  * This interface is implemented by **events** which contain ContentStreamId and NodeAggregateId.
  *
  * This is relevant e.g. for content cache flushing as a result of an event.
  *
  * @internal
+ * @deprecated Use {@see EmbedsContentStreamId} and/or {@see EmbedsNodeAggregateId} instead. Will be removed with Neos 10.
```

Comment:
its `@internal` no deprecation needed. We can remove it :)

Reply:
And Neos 9 has never been released as stable, so… 😎

Reply:
Yeah, true, true. BUT we are making the live of the beta users already hard, that's why I thought this might helpful. But yeah internal and I guess not very likely someone else used it. I'll remove it.

Reply:
Done